server/src/main/java/org/elasticsearch/common/util/BytesRefHash.java

```diff
@@ -236,4 +249,19 @@ public long ramBytesUsed() {
         return BASE_RAM_BYTES_USED + bytesRefs.ramBytesUsed() + ids.ramBytesUsed() + hashes.ramBytesUsed() + spare.bytes.length;
     }
 
+    public Finder newFinder() {
+        return new Finder();
+    }
+
+    /**
+     * Returns a finder class that can be used to find keys in the hash in a thread-safe manner
+     */
```

Comment:
seems like the comment should be on the method?

Reply:
Yep! 💯